config/jobs/kubernetes-csi/external-provisioner/external-provisioner-config.yaml

```diff
@@ -2,102 +2,6 @@
 
 presubmits:
   kubernetes-csi/external-provisioner:
-  - name: pull-kubernetes-csi-external-provisioner-1-18-on-kubernetes-1-18
```

Comment:
1.18 should be removed from gen-jobs.sh, and then all the files should be updated.

Reply:
ok